.github/workflows/update-dispatch.yml

```diff
@@ -33,8 +33,6 @@ jobs:
           - name: ${{ github.ref_name }}
             skip: true
         update:
-          - source: "versions/weekly"
-            cmd: "nix run .#scripts-repo-flake-update weekly"
```

Comment:
Why is this removed?

Reply:
We don't maintain the `versions` directory on release branches, it's all done from develop. The root of this whole problem is that we want to point at local paths on `main` where we are maintaining this stuff so we see changes in PRs but from release branches we don't want that. So each time we branch, I slowly find all the misjudged references to `./` and replace them with `github:holochain/holochain` :(

Reply:
Mmh, that seems like the answer if I had asked why you replaced `./versions/weekly` with `github:holochain/holochain?dir=versions/weekly` above. :thinking:

Reply:
Right yeah, so what this workflow does is update the inputs. E.g. if a new scaffolding had released for `weekly` and it will correctly update `./versions/weekly` but then everyone will read the version from `develop`, not the one you updated on `develop-0.4`. So it's confusing to even update it IMO

Reply:
Yes, got it.